### PR TITLE
Added check for package exist when uploading it

### PR DESCRIPTION
**Pull request by Ruslan-Aleev**
### What does it do?
- Added check for package exist when uploading it
- Moved lines to lexicons
- Fixed styles for package installer - an icon is now visible to indicate that drag and drop is working
installer_icon

**How it looks now:**
package_check

### Related issue(s)/PR(s)
https://github.com/modxcms/revolution/issues/8371
https://github.com/modxcms/revolution/pull/13851

### CODE REVIEW

File: core/lexicon/en/file.inc.php [hunk 1]
```diff
@@ -76,6 +76,9 @@
 $_lang['file_name'] = 'File Name';
 $_lang['file_quick_create'] = 'Quick Create File';
 $_lang['file_quick_update'] = 'Quick Edit File';
+$_lang['file_package_err_invalid_directory'] = 'Packages directory doesnt appear to exist!';
+$_lang['file_package_err_invalid_name'] = 'This file [[+name]] has an invalid name for the transport package';
+$_lang['file_package_err_invalid_type'] = 'This file does not appear to be a transport package';
```
Comment (mishanthrop) [inline core/lexicon/en/file.inc.php]: `doesnt` or `does not`?

Comment (JoshuaLuckers) [reply core/lexicon/en/file.inc.php]: English isn't everyone's first language.

File: core/lexicon/en/file.inc.php [hunk 2]
```diff
@@ -76,6 +76,9 @@
 $_lang['file_name'] = 'File Name';
 $_lang['file_quick_create'] = 'Quick Create File';
 $_lang['file_quick_update'] = 'Quick Edit File';
+$_lang['file_package_err_invalid_directory'] = 'Packages directory doesnt appear to exist!';
```
Comment (JoshuaLuckers) [inline core/lexicon/en/file.inc.php]:
```suggestion
$_lang['file_package_err_invalid_directory'] = 'Packages directory does not exist!';
```

File: core/lexicon/en/file.inc.php (hunk 1 above)
Comment (mishanthrop) [reply core/lexicon/en/file.inc.php]: But consistency is better language than English for UI.

File: core/src/Revolution/Processors/Workspace/Packages/Upload.php [hunk 3]
```diff
@@ -95,16 +95,25 @@ public function process()
...
-            return $this->failure("2 This file [{$file['name']}] does not appear to be a transport package"); //@TODO Make translatable
+            return $this->failure($this->modx->lexicon('file_package_err_invalid_name', [
+                'name' => $file['name']
+            ]));
+        }
+
+        $newPath = $destination . $file['name'];
+
+        // Check if a file exists
+        if (file_exists($newPath)) {
```
Comment (misha7000) [inline core/src/Revolution/Processors/Workspace/Packages/Upload.php:111]: Unnecessary comment

Comment (mishanthrop) [inline core/src/Revolution/Processors/Workspace/Packages/Upload.php:111]: Unnecessary comment

File: core/lexicon/en/file.inc.php (hunk 2 above)
Comment (opengeek) [reply core/lexicon/en/file.inc.php]: It should have an article—"The packages directory does not exist!"

**Review by opengeek — CHANGES_REQUESTED** [review]

**Review by JoshuaLuckers — APPROVED** [review]

**Review by theboxer — APPROVED** [review]

**Review by opengeek — APPROVED** [review]

**Pull request merged**